### ISSUE TITLE
Improve Discord support

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#7885]
Hello. Discord a good alternative to Skype But he has big problems with accessibility. Many people have requested the availability of discord for screen access programs But they don't do it yet. I suggest to make an add-on for NVDA or to make that with NVDA was available discord without add-ons.

**Comment by feerrenrut** [comment]
Hi, thanks for raising this feature request, as [discussed previously](https://github.com/nvaccess/nvda/pull/7120#issuecomment-304751169) the best place to request discord accessibility improvements is through their [support channel](https://support.discordapp.com).

I'll leave this issue open, if it becomes clear there is more that we can do on the NVDA side it should be recorded here.

**Comment by rjsandefur1979** [comment]
Does anybody know if discord is on gethub? If so, I wonder if it's possible to share this nvda repository with them? I'm new here.

**Comment by rjsandefur1979** [comment]
#Discord is on github. https://github.com/discordapp

**Comment by rjsandefur1979** [comment]
@ I did look and discord has 99 repositories.

**Comment by Adriani90** [comment]
According to this forum it seems Discord managed to improve accessibility in their app. Probably not everything is working but it seems it is much better:
https://forum.audiogames.net/topic/28487/discord/page/2/

Thus, I am closing this issue. Let's track the progress on that forum or directly on Redit. Please open new issues for specific problems in Discord if you think they are related to NVDA.
